ISA-XLSX.md

```diff
@@ -735,4 +783,85 @@ If we pool two sources into a single sample, we might represent this as:
+SHOULD be picked. Unregistered or niche encoding and file formats MAY be indicated instead via the most appropriate URL.
+
+The format and usage info about the Selector MAY be further qualified using a `Data Selector Format` column. The `Data Selector Format` SHOULD point to a web resource containing instructions about how the Selector is formatted and how it should be interpreted.
+
+## Explication column
+
+Every `Datamap Table sheet` SHOULD contain an `Explication` column. The `Explication` adds explicit meaning to the data node. The value MUST be free text, or an [`Ontology Annotation`](#ontology-annotations).
+
+| Explication | Term Source REF | Term Accession Number |
+|------------------------|-------------------|-------------------------|
+| average value | OBI | [http://…/obo/OBI_0000679](http://purl.obolibrary.org/obo/OBI_0000679) |
+
+## Unit column
+
+Every `Datamap Table sheet` SHOULD contain an `Unit` column. The `Unit` adds a unit of measurement to the data node. The value MUST be free text, or an [`Ontology Annotation`](#ontology-annotations).
+
+| Unit | Term Source REF | Term Accession Number |
+|------------------------|-------------------|-------------------------|
+| milligram per milliliter | UO | [http://…/obo/UO_0000176](http://purl.obolibrary.org/obo/UO_0000176) |
+
+## Object Type column
+
+Every `Datamap Table sheet` SHOULD contain an `Object Type` column. The `Object Type` defines the shape or format in which the data node is represented. The value MUST be free text, or an [`Ontology Annotation`](#ontology-annotations).
+
+| Object Type | Term Source REF | Term Accession Number |
+|------------------------|-------------------|-------------------------|
+| Float | NCIT | [http://…/obo/NCIT_C48150](http://purl.obolibrary.org/obo/NCIT_C48150) |
+
+## Description column
+
+Every `Datamap Table sheet` SHOULD contain a `Description` column. The `Description` gives additional, humand readable context about the data node. The value MUST be free text.
+
+| Description | 
+|------------------------|
+| The average protein concentration for the given gene |
+
+## Generated By column
+
+Every `Datamap Table sheet` SHOULD contain a `Generated By` column. The `Generated By` names the tool which led to the creation of the data node. The value MUST be free text.
+
+If possible, the value in this column MUST correspond to a relevant data resource location, following the [Data Path Annotation](/ARC%20specification.md#data-path-annotation) patterns.
+
+| Generated By | 
+|------------------------|
+| GeneStatisticsTool.exe |
+
+## Comments
+
+A `Comment` can be used to provide some additional information. Columns headed with `Comment[<comment name>]` MAY appear anywhere in the Annotation Table. The comment always refers to the Annotation Table. The value MUST be free text.
```

Comment:
So we must remember Comment location in ARCtrl?

Reply:
Not necessarily, as the comment column does not refere to a specific other column but always to the table as a whole.